lwallet/Pos/Wallet/Light/Redirect.hs

```diff
@@ -27,7 +26,7 @@ import qualified Pos.Wallet.Light.State    as LWS
 -- MonadBalances
 ----------------------------------------------------------------------------
 
-getOwnUtxosWallet :: (MonadIO m, Ether.MonadReader' LWS.WalletState m) => [Address] -> m Utxo
+getOwnUtxosWallet :: (MonadIO m, MonadReader ctx m, HasLens LWS.WalletState ctx LWS.WalletState) => [Address] -> m Utxo
```

Comment:
This should be replaced with `LWS.HasWalletState ctx`, I didn't remove all `HasLens` uses yet.